IFC4x3/Properties/i/IsAdjustable_00OvkE_Rn4KBS$xE3fF5xV/Documentation.md

```diff
@@ -0,0 +1 @@
+To indicate whether the dropper is adjustable or not.
```

Comment:
Either genrslise this or consider slightly more specific property name. I suggestion generic description